model/dd2d/pario_nc.f

```diff
@@ -778,8 +783,14 @@ subroutine stoprc(rc,rc_ok)
       integer :: rc,rc_ok
       integer :: mpi_err
 #ifndef SERIAL_MODE
+      call debug_msg("### Calling mpi_bcast")
       call mpi_bcast(rc,1,MPI_INTEGER,0,MPI_COMM_WORLD,mpi_err)
```

Comment:
Okay @ltmurray, this `mpi_bcast` call is where the non-root MPI ranks are hanging.